README.md

```diff
@@ -2099,7 +2099,7 @@ In order to achieve greater coverage and encourage more people to contribute to
         <tr>
             <td><a href="https://en.wikipedia.org/wiki/Linked_list">Singly Linked List</a></td>
             <td> <!-- C -->
-                <a href="./src/c/ListaEncadeada.c">
+                <a href="./src/c/LinkedList.c">
```

Comment:
```suggestion
                <a href="./src/c/SinglyLinkedList.c">
```

Reply:
I'm not sure if specificity is necessary

Reply:
I dont know if it needs to be so specific